package.json

```diff
@@ -44,13 +44,13 @@
     "ethereumjs-util": "^6.0.0",
     "ethers": "^4.0.13",
     "ethjs-abi": "^0.2.1",
-    "ganache-cli": "=6.1.8",
+    "ganache-cli": "=6.2.5",
     "solhint": "^1.2.1",
-    "truffle": "^4.1.14"
+    "truffle": "^5.0.0"
   },
   "author": "johannbarbie@me.com",
   "license": "ISC",
   "dependencies": {
-    "openzeppelin-solidity": "^1.12.0"
+    "openzeppelin-solidity": "github:OpenZeppelin/openzeppelin-solidity#release-v2.1.0-solc-0.5"
```

Comment:
i'm getting:
```
ln: illegal option -- -
usage: ln [-Ffhinsv] source_file [target_file]
       ln [-Ffhinsv] source_file ... target_dir
       link source_file target_file
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! openzeppelin-solidity@2.1.0-rc.1 compile: `scripts/compile.sh`
npm ERR! Exit status 1
```
following instructions here: https://github.com/OpenZeppelin/openzeppelin-solidity/issues/1498#issuecomment-450177442
i've executed `npm install openzeppelin-solidity@next` and it worked

Reply:
@johannbarbie 
Works for me on local too.
Travis is failing and the only difference I find suspect that travis installed (https://travis-ci.org/leapdao/solEVM-enforcer/builds/473310067#L514) more dependencies.

Maybe they do aggressive caching of npm?

Reply:
@johannbarbie Ok, I switched to the github: - scheme and it works. I think travis has some bad caching behaviour, or a kobold 😉